sdks/java/core/src/main/java/org/apache/beam/sdk/coders/CustomCoder.java

```diff
@@ -62,6 +64,22 @@
   }
 
   /**
+   * Returns an empty list. A {@link CustomCoder} may not have argument coders
```

Comment:
Updated. A `CustomCoder` doesn't have any components; it may have arguments, which map 1:1 with its generic types. (Probably needs to be more heavily documented).